Set-1/Solved Programs/BrainCar/BrainCar.java

```diff
@@ -0,0 +1,74 @@
+package com.braincar;
+
+import java.util.logging.Logger;
+
+public class BrainCar {
+	
+	private static int eachconsecutiveCount=1;
+	private static int consecutiveDifferentVar[]=new int[3];				

```

Comment:
why methods and variables are static

Reply:
If method is static no need to create object of this class while declaring this method and static variable can be used to refer the common property of all objects .